### ISSUE TITLE
Applied Accessibility - Improve Accessibility of Audio Content with the audio Element If you give audio element an id property test suite says audio doesn't have a closing tag

### ISSUE THREAD

**Issue by dverdin83** [freeCodeCamp/learn#139]
It is valid HTML to give audio an id property. It is even shown in the example On the challenge. If you give the audio attribute an id it fails the test suite saying that audio doesn't have a closing anchor.

**Comment by scissorsneedfoodtoo** [comment]
@dverdin83, thank you for pointing this out. Your issue was addressed recently over in the main freeCodeCamp repo in PR [17401](https://github.com/freeCodeCamp/freeCodeCamp/pull/17401). You shouldn't have this problem in the near future!
